Makefile

```diff
@@ -55,7 +55,9 @@ INVENTORY_SCRIPTS = \
 	support/mender-inventory-network \
 	support/mender-inventory-os \
 	support/mender-inventory-rootfs-type \
-	support/mender-inventory-geo
+
+INVENTORY_NETWORK_SCRIPTS = \
+  support/mender-inventory-geo
```

Comment:
Before Kristian comments: wrong indentation :p

Reply:
This is consistent with the other indentation :)

It's the old that was wrong ;)